packages/devtools_app/lib/src/screens/app_size/app_size_controller.dart

```diff
@@ -580,6 +582,11 @@ class AppSizeController {
       caption: isDeferred ? '(Deferred)' : null,
     )..addAllChildren(children);
   }
+
+  @visibleForTesting
+  void setDeferredLoadingSupportEnabled(bool enabled) {
```

Comment:
we don't need to add a method for this. You can just set `deferredLoadingSupportEnabled` to true directly from the test setup

Reply:
Oooh how do you mean? It is currently a top level variable in the file

Reply:
I mean in the test where you are currently calling `controller.setDeferredLoadingSupportEnabled(true);`, instead call `deferredLoadingSupportEnabled = true;`. We do this for several other feature flags in other screens.

Reply:
Ah got it, done!